_data/jobs.yml

```
@@ -1,4 +1,9 @@
-- expires: 2022-01-07
+- expires: 2022-01-30
+  location: RAND: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington D.C.
```

This `:` is probably going to give you trouble, I'd add quotes
```suggestion
  location: "RAND: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington D.C."
```